setup.py

```diff
@@ -27,9 +27,11 @@ def package_data(pkg, roots):
     version='0.3.0',
```

Comment:
Please bump the version number, and we'll tag this version too after merging.

Reply:
@swalladge  Thanks for bumping the version here!  Could you create a tag for the merged commit, named `v0.3.1` to mark the new version?

Reply:
Ah yep, done. :)